package.json

```diff
@@ -161,29 +161,29 @@
     "normalize.css": "8.0.1",
     "path": "0.12.7",
     "postcss": "8.4.31",
-    "postcss-csso": "6.0.0",
-    "postcss-discard-comments": "5.0.1",
-    "postcss-loader": "6.1.1",
-    "postcss-preset-env": "7.4.2",
+    "postcss-csso": "6.0.1",
+    "postcss-discard-comments": "6.0.0",
+    "postcss-loader": "7.3.3",
+    "postcss-preset-env": "9.2.0",
     "postcss-sass-loader": "1.1.0",
-    "prettier": "2.4.1",
+    "prettier": "3.0.3",
```

Comment:
Tried prettier v3.0.3, but reverting to last 2x version [`2.8.8`] because of conflicts with other prettier plugins.

<details><summary>Prettier 3.0.0 error</summary>

<p></p>

```bash
➜ npx gulp lintSass

[15:37:11] Using gulpfile ~/web/uswds/gulpfile.js
[15:37:11] Starting 'lintSass'...
[15:37:13] 'lintSass' errored after 2.02 s
[15:37:13] TypeError: prettier.resolveConfig.sync is not a function
    at /Users/jmejia-a/web/uswds/node_modules/stylelint-prettier/stylelint-prettier.js:50:56
    at /Users/jmejia-a/web/uswds/node_modules/stylelint/lib/lintPostcssResult.js:121:8
    at Array.map (<anonymous>)
    at lintPostcssResult (/Users/jmejia-a/web/uswds/node_modules/stylelint/lib/lintPostcssResult.js:111:18)
    at lintSource (/Users/jmejia-a/web/uswds/node_modules/stylelint/lib/lintSource.js:110:8)
    at async /Users/jmejia-a/web/uswds/node_modules/stylelint/lib/standalone.js:211:27
    at async Promise.all (index 8)
    at async Function.standalone [as lint] (/Users/jmejia-a/web/uswds/node_modules/stylelint/lib/standalone.js:254:22)
    at async lintSass (/Users/jmejia-a/web/uswds/tasks/lint.js:43:31)
```

</details>